commons/src/main/java/io/ballerina/stdlib/graphql/commons/utils/KeyDirectivesArgumentHolder.java

```diff
@@ -18,15 +18,17 @@
 
 package io.ballerina.stdlib.graphql.commons.utils;
 
+import java.io.Serializable;
 import java.util.List;
 
 /**
  * Hold the necessary arguments of GraphQL federation @key directive.
  */
-public class KeyDirectivesArgumentHolder {
+public class KeyDirectivesArgumentHolder implements Serializable {
```

Comment:
This should be added to GraalVM configs since we serialize this class.

Reply:
Added